src/less/components/steps.less

```diff
@@ -205,14 +204,14 @@
             text-align: left;
             padding: @list-item-padding-desktop-vertical;
 
-            &.steps-selected {
-                box-shadow: 0 1px 0 0 @gray-bright, 0 -1px 0 0 @gray-bright;
-            }
-
             > a {
                 padding: @list-item-padding-desktop-vertical;
                 width: auto;
                 .negative-margin(@list-item-padding-desktop-vertical);
+
+                &.steps-selected {
```

Comment:
Why have you moved the `.steps-selected` class from the `li`element to the anchor element here?

Reply:
..just an error. Fixed it and double checked it, should be okay now!